README.md

```diff
@@ -360,6 +369,87 @@ Calculates the basic amount of gain for a non-resident capital gains user
         "prrUsed":44.0
     }
 }
+
+## GET /capital-gains-calculator/non-resident/calculate-tax-owed
```

Comment:
The json response before needs to be closed so this title has the correct formatting.